MSAL/src/MSALInteractiveTokenParameters.m

```diff
@@ -27,12 +27,18 @@
 
 #import "MSALInteractiveTokenParameters.h"
 #import "MSALTokenParameters+Internal.h"
-
+#import "MSALGlobalConfig.h"
 @implementation MSALInteractiveTokenParameters
 
 - (instancetype)initWithScopes:(NSArray<NSString *> *)scopes
 {
-    return [super initWithScopes:scopes];
+    self = [super initWithScopes:scopes];
+    if (self)
+    {
+        _webviewType = MSALGlobalConfig.defaultWebviewType;
+        _customWebview = nil;
```

Comment:
nit: would custom web view be not nil?

Reply:
it'll be nil, but doesn't hurt to be specific :)

Reply:
Will remove though, nothing bad comes by reducing lines of code when they are not needed